x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -614,6 +653,51 @@ export const AlertsList: React.FunctionComponent = () => {
         );
       },
     },
+    {
+      field: 'monitoring.execution.calculated_metrics.p50',
+      width: '8%',
+      name: renderPercentileTableCell(Percentiles.P50),
+      'data-test-subj': 'alertsTableCell-ruleExecutionPercentile',
+      sortable: true,
+      truncateText: false,
+      render: (value: number) => {
```

Comment:
I feel like this should be a reusable function called `renderPercentileTableCell` and we should change the other one to `renderPercentileTableName` or something.

Reply:
good call